src/services/InputTransactorService.ts

```diff
@@ -97,22 +124,20 @@ export default class InputTransactorService {
 
     static getNonce = async (senderAccount: string, connectedChainId: string) => {
         try {
-            const config: Record<string, any> = configFile
-            const url = `${config[connectedChainId].graphqlAPIURL}/graphql`;
-            const query = `
-            {inputs(where: {msgSender: "${senderAccount}" type: "Espresso"}) {
-                totalCount
-        }}`;
-            const response = await fetch(url, {
-                method: 'POST',
+            const config = {
                 headers: {
                     'Content-Type': 'application/json'
-                },
-                body: JSON.stringify({ query })
-            });
+                }
+
+            }
+            const data = {
+                app_contract: "0xab7528bb862fB57E8A2BCd567a2e929a0Be56a5e",
+                msg_sender: senderAccount
+            }
+            const response = await axios.post("http://localhost:8080/nonce", data, config);
+
 
-            const responseData = await response.json();
-            const nextNonce = responseData.data.inputs.totalCount + 1;
+            const nextNonce = response.data.nonce + 1;
```

Comment:
I believe this sum or whatever logic we do is going to stay on the backend. I'll check.